PatientRegistrationApp/src/components/SinglePatientPersonalInfo.vue

```diff
@@ -76,6 +75,7 @@
           :rules="rules"
           accept="image/png, image/jpeg, image/bmp"
           placeholder="Upload a recent photo"
+          v-model="recentPhoto"
```

Comment:
I would make this patientPhoto. "Recent" doesn't really have a bearing on the data itself

Reply:
Updated accordingly